src/main/scala/cromwell/binding/RuntimeAttributes.scala

```diff
@@ -79,6 +82,10 @@ object RuntimeAttributes {
     value map validateBoolean getOrElse Defaults.FailOnStderr.successNel
   }
 
+  private def validateFailOnRc(value: Option[String]): ValidationNel[String, Boolean] = {
+    value map validateBoolean getOrElse Defaults.FailOnRc.successNel
```

Comment:
(Probably misreading this), why are these all returning success if the validation fails?